### DISCUSSION TITLE
加入返回旧版功能

### DISCUSSION THREAD

**Discussion by FCSDFSDVXDFG** [the1812/Bilibili-Evolved#3767] [功能建议 / Ideas]
新版的视频网页冲击我的审美，而且返回旧版按钮还被砍掉了，希望可以添加回来

**Comment by the1812** [comment]
建议去 https://github.com/MotooriKashin/Bilibili-Old 问

**Reply by FCSDFSDVXDFG** [reply]
并不是要求复刻早期的界面
https://www.bilibili.com/video/BV1od4y1z7KD/

**Reply by the1812** [reply]
这种做出来可能过段时间就失效了, 手动改改就算了
